modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/HashAggregatePlannerTest.java

```diff
@@ -42,6 +46,77 @@
  */
 @SuppressWarnings({"TooBroadScope", "FieldCanBeLocal", "TypeMayBeWeakened"})
 public class HashAggregatePlannerTest extends AbstractAggregatePlannerTest {
+    /**
+     * Tests COUNT(...) plan with and without IndexCount optimization.
+     */
+    @Test
+    public void indexCount() throws Exception {
+        IgniteSchema publicSchema = new IgniteSchema("PUBLIC");
+        TestTable tbl = createBroadcastTable();
```

Comment:
Other distributions should be tested too

Reply:
Fixed